src/template/partials/footer.html

```diff
@@ -0,0 +1,62 @@
+<footer class="footer-main">
+    <div class="container">
+        <div class="row">
+            <div class="col-12 col-lg-4">
+                <div
+                    class="stract-footer d-flex flex-column justify-content-center"
+                >
+                    <img src="assets/images/logo-footer.svg" alt="" />
+                    <span
+                        >Data analysis software is a type of software tool used
+                        for data analysis and reporting.</span
+                    >
+                    <div class="icon-footer d-flex align-items-center">
+                        <a><i class="fa-brands fa-twitter"></i></a>
+                        <a><i class="fa-brands fa-facebook-f"></i></a>
+                        <a><i class="fa-brands fa-instagram"></i></a>
+                        <a><i class="fa-brands fa-github"></i></a>
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 offset-lg-1 col-lg-7">
+                <div class="row">
+                    <div class="col-12 col-lg-2">
+                        <div class="list-footer">
+                            <h6>Company</h6>
+                            <ul>
+                                <li><a href="">Service</a></li>
+                                <li><a href="">Resources</a></li>
+                                <li><a href="">About us</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="col-12 col-lg-3">
+                        <div class="list-footer">
+                            <h6>Help</h6>
+                            <ul>
+                                <li><a href="">Customer Support</a></li>
+                                <li><a href="">Terms & Conditions</a></li>
+                                <li><a href="">Privacy Policy</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="col-12 col-lg-7">
+                        <div class="list-footer">
+                            <h6>Subscribe to Newsletter</h6>
+                            <form class="subscribe d-flex align-items-center">
+                                
+                                <input type="text" placeholder="Enter email address" />
+                                <button>Join</button>
```

Comment:
@shaban011147 @M-Khairy-B The only place you're allowed to add the button type submit, you didn't 😒

Reply:
Done